dev/io.openliberty.jakarta.security.4.0/bnd.bnd

```diff
@@ -0,0 +1,46 @@
+#*******************************************************************************
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 17
+javac.target: 17
+
+Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=17))"
+
+Bundle-SymbolicName: io.openliberty.jakarta.security.4.0; singleton:=true
```

Comment:
If this is a new bundle, which feature is it part of? Doesn't appear to have been added to one as part of this PR.

Reply:
Yes this project is completely unnecessary since the io.openliberty.jakarta/security.4.0.bnd file already exists.  If you need to update this bundle, you would update that file and not this one.  The excludes in the `includeresource` instruction in your bnd file that are extra do not exist in the api jar from jakarta so I don't think there is anything needed to be changed even in that file.